alvr/graphics/src/lobby.rs

```diff
@@ -504,6 +504,66 @@ impl LobbyRenderer {
                 transform_draw(&mut pass, view_proj * transform, 4);
             }
 
+            fn draw_crosshair(
```

Comment:
Should be "crossair" without "h". Actually it should be cross-air, but when there is a hyphenated word i usually remove the hyphen in between when using snake_case

Reply:
Oh damn, you're right. it's actually a single word, not cross-hair lol. @curoviyxru let's revert this back to crosshair

Reply:
The etymology is actually correct for "hair", where hair, spiderwebs or wires were used for the scopes